Gruntfile.js

```diff
@@ -1,10 +1,19 @@
 'use strict';
 
+var commandPath;
+
+if (process.platform === "win32") {
+    commandPath = ".\\node_modules\\.bin\\component install";
+} else {
+    commandPath = "./node_modules/.bin/component install";
+}
```

Comment:
Ah, yeah I guess I kinda neglected Windows when I wrote this code. Note that this hack with `grunt-shell` was only needed because we're still using Component. Ideally we should eliminate the need for this OS-specific code entirely. See #188.